articles/active-directory/app-provisioning/inbound-provisioning-api-issues.md

```diff
@@ -40,10 +40,13 @@ This document covers commonly encountered errors and issues with inbound provisi
 
 **Probable causes**
 1. Your API-driven provisioning app is paused. 
-1. The provisioning service is yet to update the provisioning logs with the bulk request processing details. 
+1. The provisioning service is yet to update the provisioning logs with the bulk request processing details.
+2. Your On-premises provisioning agent status is inactive ( If you are running the API-driven inbound user provisioning to on-premises AD) [https://learn.microsoft.com/en-us/azure/active-directory/app-provisioning/inbound-provisioning-api-configure-app]. 
```

Comment:
1. Remove the extra space between "(" and "If"
2. Make sure there is no space between "on-premise AD)" and "[https://,..." 
3. Replace the URL here with this go-to link: **https://go.microsoft.com/fwlink/?linkid=2245182**
(learn.microsoft.com links aren't allowed).